google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -639,6 +639,9 @@ def rotate encryption_key: nil, new_encryption_key: nil
         # Permanently deletes the file.
         #
         # @return [Boolean] Returns `true` if the file was deleted.
+        # @param [Boolean,Integer] generation Select a specific revision of the
+        #   file to delete. When `true` it will use the object's generation. The
```

Comment:
OK, I understand now. How about this:

> Specify a version of the file to delete. When `true`, it will delete the version returned by {#generation}. The default behavior is to delete the latest version of the file (regardless of the version returned by {#generation}.)